ittapi-rs/ittapi/src/jit.rs

```diff
@@ -0,0 +1,175 @@
+//! The JIT (Just-In-Time) Profiling API provides functionality to report information about
```

Comment:
Confused here. In a demo you showed the JIT code was not visible. Is this code simply not called/hooked in yet; why does the current version not resolve JIT if this is here?

Reply:
Yeah, this isn't wired up to Sightglass yet. But this code is what @bnjbvr wants to replace the code in [`vtune_linux.rs`](https://github.com/bytecodealliance/wasmtime/blob/2649d2352c415979ea0fe7f43eacb2f3f65d771c/crates/jit/src/profiling/vtune_linux.rs).